package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@canonical/jujulib": "4.0.0",
     "@canonical/macaroon-bakery": "1.3.2",
-    "@canonical/react-components": "0.44.0",
+    "@canonical/react-components": "0.45.0",
```

Comment:
I forgot we also needed to merge main into the feature branch to get this update.